2DPlatformerController/Assets/TimeScript.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class TimeScript : MonoBehaviour
+{
+    public float timeStart;
+    public Text textBox;
+    public static int numberOfCollectables;
```

Comment:
1. Initialize these fields
2. These should be private if nothing else accesses them (or should access them)
3. numberOfCollectables does not need to be static. It can be a const if you like, in which case it should be initialized at the desired number. Also collectibles is spelled incorrectly.